bin/benchpark

```diff
@@ -282,14 +284,30 @@ def benchpark_setup_handler(args):
         )
 
         run_command(f"{ramble_exe} repo add --scope=site {source_dir}/repo")
+        run_command(f"{ramble_exe} config --scope=site add \"config:disable_progress_bar:true\"")
+        run_command(f"{ramble_exe} config --scope=site add \"config:spack:global:args:'-d'\"")
```

Comment:
>> Will setting this env var make Ramble call spack with -d?

> There is currently not an environment variable that does this in Spack (I can see the appeal though).

It was pointed out to me that while this isn't possible in Spack, Ramble has a means for this